src/platform/graphics/render-pass.js

```diff
@@ -176,6 +176,9 @@ class RenderPass {
         this._execute = execute;
     }
 
+    destroy() {
```

Comment:
Are you expecting to start deallocating stuff in the base class here? If not, I'm not sure I see the value of an empty `destroy` function.

Reply:
Yes I am looking at managing render target allocations and resizing in the base class.